include/LMS1xx/colaa.h

```diff
@@ -0,0 +1,268 @@
+#ifndef COLAA_H
```

Comment:
Do we really need this? the package.xml already specifies it's LGPL and authorship is already tracked by git.